src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1635,4 +1635,13 @@
   <data name="DoNotUseWhenAllWithSingleTaskFix" xml:space="preserve">
     <value>Replace 'WhenAll' call with argument</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="PreferHashDataOverComputeHashAnalyzerTitle" xml:space="preserve">
+    <value>Prefer static 'HashData' method over 'ComputeHash'</value>
+  </data>
+  <data name="PreferHashDataOverComputeHashAnalyzerDescription" xml:space="preserve">
+    <value>It is simpler to use the static 'HashData' method over creating and managing a HashAlgorithm instance to call 'ComputeHash'.</value>
```

Comment:
Given this is an analyzer with "Performance" category, I think the messaging should be slightly different.
```suggestion
    <value>It is more efficient to use the static 'HashData' method over creating and managing a HashAlgorithm instance to call 'ComputeHash'.</value>
```